### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'href')

### ISSUE THREAD

**Issue by vawogbemi** [jedisct1/libsodium.js#323]
getting this error when I'm running _sodium.ready in a cloudflare worker

**Comment by vawogbemi** [comment]
ended up fixing it through #212, but now I'm getting this 

<img width="574" alt="image" src="https://github.com/jedisct1/libsodium.js/assets/21068868/95a07b82-1a3c-45f7-997d-74fad7dd85a1">

**Comment by vawogbemi** [comment]
@jedisct1 Hey is there any chance you have a definition of crypto_pwhash i can copy paste? Or is there another library that has a similar function.

**Comment by jedisct1** [comment]
Are you using the `sumo` version? Are other functions accessible?

If this is too complicated, you can simply use the WebCrypto API: https://bradyjoslin.com/blog/encryption-webcrypto/

**Comment by vawogbemi** [comment]
yeah im using sumo, cloudflare workers download the package directly and paste the code in the cloud env, so that may be why

**Comment by vawogbemi** [comment]
the other functions are working though
